TestCntlrApp/src/tfwApp/fw_api_int.c

```diff
@@ -3131,6 +3131,7 @@ PRIVATE Void handleEsmInformationRsp(ueEsmInformationRsp_t* data)
    uetMsg->msgType = UE_ESM_INFORMATION_RSP_TYPE;
    ueEsmInfoRsp = &uetMsg->msg.ueEsmInformationRsp;
    ueEsmInfoRsp->ueId    = data->ue_Id;
+   ueEsmInfoRsp->tId     = data->tId;
```

Comment:
Nit: too many spaces before "=". Single space before and after.

Reply:
done